test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -39,38 +39,38 @@
 
     @Before
     public void setUp() throws Exception {
-        tagPrizeWinner   = new Tag("prizewinner");
-        tagScientist     = new Tag("scientist");
+        tagPrizeWinner = new Tag("prizewinner");
+        tagScientist = new Tag("scientist");
         tagMathematician = new Tag("mathematician");
-        tagEconomist     = new Tag("economist");
+        tagEconomist = new Tag("economist");
 
-        aliceBetsy     = new Person(new Name("Alice Betsy"),
-                                    new Phone("91235468", false),
-                                    new Email("alice@nushackers.org", false),
-                                    new Address("8 Computing Drive, Singapore", false),
-                                    new UniqueTagList(tagMathematician));
+        aliceBetsy = new Person(new Name("Alice Betsy"),
+                new Phone("91235468", false),
```

Comment:
Do note that if you are separating the parameters into multiple lines, the parameters should be aligned with each other. Sometimes the formatting style in IntelliJ is slightly different from what we expect.